sageone_data.php

```diff
@@ -1,11 +1,17 @@
 <!DOCTYPE html>
 <?php
-$access_token = $_GET['access_token'];
+$response = $_GET['token_response'];
+$access_token = json_decode($response, true)['access_token'];
+/* prettify JSON response for readability */
+$json = json_decode($response);
+$pretty_json = json_encode($json, JSON_PRETTY_PRINT);
 ?>
 <html>
   <head>
     <title>Authorize with Sage One</title>
     <link type="text/css" rel="stylesheet" href="sample_app.css">
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
? http://code.jquery.com/jquery-latest.min.js ?
or do we need a fixed version?
😎